msteams-platform/toolkit/add-resource.md

```diff
@@ -46,7 +46,7 @@ You can add cloud resources in the following ways:
 
       :::image type="content" source="~/assets/images/teams-toolkit-v2/manual/cloud/Teams-add-features_1.png" alt-text="Screenshot shows the Command Palette option under View.":::
 
-   1. Enter **How-to Guides** and select **Teams:View How-to Guides** from the list.
+   1. Enter **How-to Guides** and select **Microsoft 365 Agent: View How-to Guides** from the list.
```

Comment:
Make it consistent at all the places: 
![image](https://github.com/user-attachments/assets/a75b2888-4672-48b2-b607-cf26ced06967)